packages/api-reference/src/components/ApiReferenceLayout.vue

```diff
@@ -317,7 +318,9 @@ const themeStyleTag = computed(
     </div>
     <!-- Rendered reference -->
     <template v-if="showRenderedContent">
-      <div class="references-rendered">
+      <section
```

Comment:
@kayla-beckham since the references are often embedded in a page with other content this has to be an aria `region` landmark rather than a `main` landmark otherwise we could end up with nested `main` landmarks which is a WCAG 🙅‍♀️ 

I added a `<main>` tag to the standalone CDN version of the references which is meant to be alone on the page but if have the references embedded you'll need to add your own `main` landmark somewhere.